### DISCUSSION TITLE
cli test failing with go stack

### DISCUSSION THREAD

**Discussion by gitssinghal** [cockroachdb/cockroach#62437] [Q&A] [answered]
Hi There,

I am trying building cockroachdb with below env
cockroachdb - 20.1.0
go - 1.13.5
on ppc.

While apart from 2 test suits all passes the cli one fails exactly with the same error for all version of Go.

**This is how it starts**

--- PASS: TestZipContainsAllInternalTables (0.14s)
=== RUN   TestZip
W210323 07:32:38.999023 21457095 server/status/runtime.go:308  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
W210323 07:32:43.425495 21457223 kv/kvserver/store.go:1541  [n1,s1,r6/1:/Table/{SystemCon…-11}] could not gossip system config: [NotLeaseHolderError] r6: replica (n1,s1):1 not lease holder; lease holder unknown
W210323 07:32:43.478680 21457223 kv/kvserver/store.go:1541  [n1,s1,r6/1:/Table/{SystemCon…-11}] could not gossip system config: [NotLeaseHolderError] r6: replica (n1,s1):1 not lease holder; lease holder unknown
W210323 07:32:43.566640 21457223 kv/kvserver/store.go:1541  [n1,s1,r6/1:/Table/{SystemCon…-11}] could not gossip system config: [NotLeaseHolderError] r6: replica (n1,s1):1 not lease holder; lease holder unknown
[debug --host=127.0.0.1:42153 --insecure=false --certs-dir=/tmp/cli-test139692508 zip /dev/null]
fatal: morestack on g0
fatal: morestack on g0
SIGSEGV: segmentation violation
PC=0x102c1000 m=0 sigcode=1

goroutine 0 [idlefatal: morestack on g0
]:
runtime.abort()
	/usr/local/go/src/runtime/asm_ppc64x.s:824
runtime.morestack()
	/usr/local/go/src/runtime/asm_ppc64x.s:320 +0x18

There is more on the failure trail. If required I will paste it.

All help on this appreciated.

Regards,
Swati

**Comment by knz** [comment, marked as answer]
Please file a regular issue with the full stack trace. This looks like a regular bug we need to look into.

@jordanlewis how do you suggest we can make it clear in this "discussions" feature that regular bugs should be filed as issues instead of q&a?
